### DISCUSSION TITLE
What is the best solution for Git+Shopify CI-CD

### DISCUSSION THREAD

**Discussion by moseshope** [community/community#40770] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello Git community cybers.
Today I wanna discuss on this topic for Shopify + Github.
What is the best solution?
Hope good recommendations.

**Comment by ossanna16** [comment, marked as answer]
Hi there @moseshope and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by moseshope** [reply]
Thanks for your warm answering.
Got it what you are teaching in fact.
This really helped me to proceed my development as well.
May I ask any question further if meet any touble regading to github using and problem solving?
If so, it would be great for me.
Thank you very much gain.
:)

**Comment by moseshope** [comment]
Thanks @ossanna16 to push me up to proceed my work on GitHub + Shopify CI-CD establishment.
Great work.
